internal/expanders/gatekeeper.go

```diff
@@ -110,7 +110,7 @@ func (g GatekeeperPolicyExpander) Expand(
 								"constraint_action": "deny",
 								"constraint_kind":   constraintKind,
 								"constraint_name":   constraintName,
-								"event_type":        "violation",
+								"event_type":        "violation_audited",
 							},
 						},
 					},
```

Comment:
The simplest thing might be to just add it as another `object-template`. What would you think of that, @itewk?
```suggestion
					},
					{
						"complianceType": "mustnothave",
						"objectDefinition": map[string]interface{}{
							"apiVersion": "v1",
							"kind":       "Event",
							"annotations": map[string]interface{}{
								"constraint_action": "deny",
								"constraint_kind":   constraintKind,
								"constraint_name":   constraintName,
								"event_type":        "violation",
								"event_type":        "violation_audited",
							},
						},
					},
```

Reply:
yeah. but im also fighting with the hard coded namespaceSelector and the note left behind about "futer improvements needed". its annoying having namepsaces /excludedNamespaces set in my constraint (both at the gatekeeper and OPM level) but then those don't propigate to the generated ConfigurationPolicy(s) which ar harded cluded eclude/includes.

i am going to keep playing with it to see if i can get the behavor i would expect.